netsgiro/__init__.py

```diff
@@ -1 +1,27 @@
+from typing import List
+
+from netsgiro import enums, records
+
+
 __version__ = '0.1.0'
+
+
+# Nets' data sender/recipient ID
+NETS_ID = '00008080'
+
+
+def get_records(data: str) -> List[records.Record]:
+    lines = data.strip().splitlines()
+
+    for line in lines:
+        if len(line) != 80:
+            raise ValueError('All lines must be exactly 80 chars long')
+
+    results = []
+
+    for line in lines:
+        record_type = enums.RecordType(int(line[6:8]))
```

Comment:
extract specifier and add explicit test for whether it's a valid record type. 
One for whether its well formed (`isnumber()`), with a helpful error message,
and one for whether its a supported type (`specifier not in records.RECORD_CLASSES`)